### DISCUSSION TITLE
Create User via API

### DISCUSSION THREAD

**Discussion by binhodev** [mediacms-io/mediacms#631] [Feature Requests]
**Describe the feature you'd like**
The possibility of creating users using the REST API, since it is possible to edit the information and delete the user's account.

You have done a beautiful job.

**Comment by hojjatzangeneh** [comment]
it's good idea and i need this api for my site

**Comment by hojjatzangeneh** [comment]
i install django-rest-allauth and work for me

**Comment by binhodev** [comment]
Mmmm, how to this works? Django don't my beach.

**Comment by edmsamuel** [comment]
Hello, can someone do this?
